src/Faker/Provider/es_CL/Internet.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Faker\Provider\es_CL;
+
+class Internet extends \Faker\Provider\Internet
+{
+    protected static $freeEmailDomain = array('gmail.com', 'hotmail.com', 'hotmail.cl', 'yahoo.com', 'yahoo.cl', 'live.com', 'live.cl','vtr.net', 'latinmail.com', 'terra.com','gob.cl');
+    protected static $tld = array('com', 'com', 'com', 'com', 'net', 'org', 'org', 'cl','cl','cl','gob.cl');
+
+    /**
+     * Converts Spanish characters to their ASCII representation using an standard
+     * chars convert function
+     *
+     * @return string
+     */
+    private static function toAscii($string)
```

Comment:
deprecated. See the original Internet provider, it now has a built-in translitterator